common/src/main/java/com/duckduckgo/app/global/device/Device.kt

```diff
@@ -41,11 +39,16 @@ interface DeviceInfo {
     fun formFactor(): FormFactor
 }
 
-class ContextDeviceInfo @Inject constructor(private val context: Context) : DeviceInfo {
+class ContextDeviceInfo(private val context: Context) : DeviceInfo {
 
-    override val appVersion = BuildConfig.VERSION_NAME
+    override val appVersion by lazy {
+        val info = context.packageManager.getPackageInfo(context.packageName, 0)
+        info.versionName.orEmpty()
+    }
```

Comment:
`BuildConfig.VERSION_NAME` cannot be access from a lib gradle module. So we get the version name programatically